_unittest/conftest.py

```diff
@@ -94,7 +94,7 @@
 desktop_version = config["desktopVersion"]
 new_thread = config["NewThread"]
 settings.use_grpc_api = config["use_grpc"]
-
+settings.objects_lazy_load = False
```

Comment:
Is it not tested? I think we should run at least Primitives test with and without lazy load